common/environment/environment.cpp

```diff
@@ -31,11 +31,77 @@
 #include "dalienv.hpp"
 
 #define SDS_LOCK_TIMEOUT  30000
-
+#define DEFAULT_DROPZONE_INDEX      1
+#define DROPZONE_BY_MACHINE_SUFFIX  "-dropzoneByMachine-"
+#define DROPZONE_SUFFIX             "dropzone-"
+#define MACHINE_PREFIX              "machine-"
+#define NUMBER_OF_DROPZONES_SUFFIX  "-numberOfDropZones"
 
 static int environmentTraceLevel = 1;
 static Owned <IConstEnvironment> cache;
 
+class CLocalEnvironment;
+
+class CConstMachineInfoIterator : public CInterface, implements IConstMachineInfoIterator
+{
+public:
+    CConstMachineInfoIterator();
+    IMPLEMENT_IINTERFACE
+    bool first();
+    bool next();
+    bool isValid();
+    virtual IConstMachineInfo & query();
+    virtual unsigned count();
+
+protected:
+    IConstMachineInfo * curr;
+    Owned<CLocalEnvironment> constEnv;
+    unsigned index;
+    unsigned maxIndex;
+};
```

Comment:
getMachineIterator() is not used by PR so nor is this class.
I think it better removed unless you know what will use it for the purposes of this issue.